tests/V1/incident/test_source_system.py

```diff
@@ -0,0 +1,128 @@
+from django.contrib.auth import get_user_model
+from django.test import Client, tag, override_settings
+from django.urls import reverse
+
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APIClient, APITestCase
+
+from argus.auth.factories import AdminUserFactory
+from argus.incident.factories import SourceSystemTypeFactory
+from argus.incident.models import SourceSystem
+
+
+User = get_user_model()
+
+
+@tag("api", "integration")
+@override_settings(ROOT_URLCONF="tests.V1.v1_root_urls")
+class SourceSystemPostingTests(APITestCase):
+    def setUp(self):
+        self.type1 = SourceSystemTypeFactory(name="nav")
+        self.type2 = SourceSystemTypeFactory(name="zabbix")
+
+        password = "1234"
+        self.user1 = AdminUserFactory(username="user1", password=password)
+        user_token = Token.objects.create(user=self.user1)
+
+        self.rest_client = APIClient()
+        self.rest_client.credentials(HTTP_AUTHORIZATION=f"Token {user_token.key}")
+        self.django_client = Client()
+        self.django_client.login(username=self.user1.username, password=password)
+
+        # URL format: https://docs.djangoproject.com/en/4.2/ref/contrib/admin/#reversing-admin-urls
```

Comment:
Since #941 this needs to be changed to

```suggestion
        # URL format: https://docs.djangoproject.com/en/5.2/ref/contrib/admin/#reversing-admin-urls
```